brave-intl/publishers
# Update Referral Page on Admin

## Update Referral Page on Admin

![localhost_3000_admin_publishers_e78e8145-9ea3-45b9-abab-3cd5dffded34_referrals](https://user-images.githubusercontent.com/5459225/68423123-1f809080-0167-11ea-95cf-995459592030.png)

#### Features

- Add last updated field to promo panel
- Add ability to view groupings in admin view

#### How To Test

- Mock data for promo stats by running `rails database_updates:mock_data:populate_promo_stats`
- Login to admin and open up the referrals.
- Ensure data is present and you can change the month
